Add binary.dart

```diff
@@ -0,0 +1,37 @@
+//Qa(67)
+class Solution {
+  String addBinary(String a, String b) {
+    if (a.length != b.length) {

```

Comment:
bare in mind that you're calculating the length each time you call a.length or b.length
it is always better to calculate once and use in the context later like:
```
int aLength = a.length;
int bLength = b.length;
// and later 
if (aLength > bLength) { ... }
```